card/src/main/res/values/strings.xml

```diff
@@ -32,6 +32,7 @@
 
     <string name="checkout_card_installments_title">Number of installments</string>
     <string name="checkout_card_installments_option_regular">%s months</string>
+    <string name="checkout_card_installments_option_regular_with_price">%1$sx %2$s</string>
```

Comment:
We should move this to the bottom of the file and add `translatable="false"` to it, then do the exact same change in `strings.xml.tt`. This is necessary for our translations process.

Reply:
I left it here since I was not sure if in some locales they use the multiplies differently. E.g. `%2$s x%1$s`.
I will check how other teams handle this. If this does not make sense to localize, I will add `translatable="false"` and move it to the bottom of the file.

Thanks for pointing this out!